spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -324,44 +402,64 @@ def call_with_params(params)
       end
 
       context "with a valid owner tag" do
+        let!(:fandom2) { create(:canonical_fandom) }
+        let!(:work2) { create(:work, fandom_string: fandom2.name) }
+
         before do
-          @fandom2 = create(:canonical_fandom)
-          @work2 = create(:work, fandom_string: @fandom2.name)
           run_all_indexing_jobs
         end
 
         it "only gets works under that tag" do
-          get :index, params: { tag_id: @fandom.name }
-          expect(assigns(:works).items).to include(@work)
-          expect(assigns(:works).items).not_to include(@work2)
+          get :index, params: { tag_id: fandom.name }
+          expect(assigns(:works).items).to include(work)
+          expect(assigns(:works).items).not_to include(work2)
         end
 
         it "shows different results on second page" do
-          get :index, params: { tag_id: @fandom.name, page: 2 }
-          expect(assigns(:works).items).not_to include(@work)
+          get :index, params: { tag_id: fandom.name, page: 2 }
+          expect(assigns(:works).items).not_to include(work)
+        end
+
+        context "when suspend_filter_counts is on" do
+          before do
+            allow(controller).to receive(:fetch_admin_settings).and_return(true)
+            AdminSetting.first.update_attribute(:suspend_filter_counts, true)
+            admin_settings = AdminSetting.first
+            controller.instance_variable_set("@admin_settings", admin_settings)
+          end
+
+          after do
+            allow(controller).to receive(:fetch_admin_settings).and_call_original
+          end
+
+          it "shows results when filters are disabled" do
```

Comment:
Let's describe this as `it "returns the work" do`. That will

* match the description of the related example on line 332
* get rid of the incorrect part about filters being disabled (we're testing when filter counts are suspended, which is made clear by the fact this is inside `context "when suspend_filter_counts is on" do`)

Reply:
I've tweaked slightly further to "shows the work in the index", and also clarified the description a couple of lines down.